README.md

```diff
@@ -1,108 +1,109 @@
+   systems with Python 3.n or Python 2.7, these commands are
 
-    ```
-    mkdir build
-    cd build
-    cmake ..
-    make -j <number_of_processor_cores_on_system>
-    make install 
-    pip install -U python/dist/ngraph-0.5.0-py2.py3-none-linux_x86_64.whl
-    ```
+        virtualenv --system-site-packages -p python3 your_virtualenv 
+        virtualenv --system-site-packages -p /usr/bin/python2 your_virtualenv  
+        source your_virtualenv/bin/activate # bash, sh, ksh, or zsh
+    
+2. Checkout `v0.5.0` from the `ngraph-tf` repo and build the bridge
+   as follows: 
+   
+        git clone https://github.com/NervanaSystems/ngraph-tf.git
+        cd ngraph-tf
+        git checkout v0.5.0
+        mkdir build
+        cd build
+        cmake ..
+        make -j <number_of_processor_cores_on_system>
+        make install 
+        pip install -U python/dist/ngraph-0.5.0-py2.py3-none-linux_x86_64.whl
 
-## Option 2: Build nGraph bridge from source using TensorFlow source
 
-If you want to run unit tests or are planning to contribute, then you need to install 
-the nGraph bridge using the TensorFlow source tree. 
+### Option 2: Use the Build nGraph bridge from source using TensorFlow source
 
-### Prepare the build environment
+To run unit tests, or if you are planning to contribute, install the nGraph 
+bridge using the TensorFlow source tree. 
 
-The installation prerequisites are the same as TensorFlow as described in the 
-TensorFlow [prepare environment] for linux.
+#### Prepare the build environment
+
+The installation prerequisites are the same as described in the TensorFlow 
+[prepare environment] for linux.
 
 1. We use the standard build process which is a system called "bazel". These 
    instructions were tested with [bazel version 0.16.0]. 
 
-   ```
-   $ wget https://github.com/bazelbuild/bazel/releases/download/0.16.0/bazel-0.16.0-installer-linux-x86_64.sh      
-   $ chmod +x bazel-0.16.0-installer-linux-x86_64.sh
-   $ ./bazel-0.16.0-installer-linux-x86_64.sh --user
-   ```
+        wget https://github.com/bazelbuild/bazel/releases/download/0.16.0/bazel-0.16.0-installer-linux-x86_64.sh      
+        chmod +x bazel-0.16.0-installer-linux-x86_64.sh
+        ./bazel-0.16.0-installer-linux-x86_64.sh --user
 
 2. Add and source the ``bin`` path to your ``~/.bashrc`` file in order to be 
    able to call bazel from the user's installation we set up:
 
-   ```  
-   export PATH=$PATH:~/bin
-   $ source ~/.bashrc   
-   ```
+        export PATH=$PATH:~/bin
+        source ~/.bashrc   
 
-3. Ensure that all the TensorFlow dependencies are installed, as per the
-   TensorFlow [prepare environment] for linux.
+3. Ensure that all the TensorFlow dependencies are installed, and when building 
+   TensorFlow* *do not* select `Yes` when asked: 
+        Do you wish to build TensorFlow with CUDA support? [y/N]: N
+        No CUDA support will be enabled for TensorFlow.
 
```

Comment:
When these lines are visualized in chrome, they stick together and it looks like:
" ...CUDA support? [y/N]: N No CUDA support ..."

The "N No" looks weird. We should put the "Do you wish to build TensorFlow with CUDA support? [y/N]: N" in code format maybe?

Reply:
Thanks, was just missing a blank line.